v2/src/core/plugins/vuetify.ts

```diff
@@ -74,8 +77,10 @@ const mainnetDarkTheme: ThemeDefinition = {
         'on-surface': COLORS.lightBlue,
         'on-primary': COLORS.lightBlue,
         'on-secondary': COLORS.lightBlue,
+        'on-success': COLORS.surfaceDM,
         'on-surface-tabs': COLORS.lightBlue,
-        pillGrey: COLORS.greyBackdroundDM
+        pillGrey: COLORS.greyBackdroundDM,
+        tableGrey: COLORS.greyBackdroundDM
```

Comment:
where are we using this color?

Reply:
That's the background color when you expand a table row